poc_iot_injector/src/server.rs

```diff
@@ -1,28 +1,46 @@
-// TODO: Remove
-#![allow(dead_code, unused)]
-
-use crate::{follower::FollowerService, keypair::Keypair, Result};
-// use helium_proto::{
-//     blockchain_txn::Txn,
-//     BlockchainTxn, Message, BlockchainTxnPOCReceiptsV2
-// };
-// use poc_metrics::record_duration;
-// use poc_store::FileStore;
+use crate::{
+    error::DecodeError, keypair::Keypair, receipt_txn::handle_report_msg,
+    txn_service::TransactionService, Result,
+};
+use chrono::{DateTime, NaiveDateTime, TimeZone, Utc};
+use db_store::MetaValue;
+use file_store::{FileStore, FileType};
+use futures::stream::{self, StreamExt};
+use helium_proto::{blockchain_txn::Txn, BlockchainTxn};
+use sqlx::{Pool, Postgres};
+use std::env;
 use tokio::time;
 
 // 30 mins
 pub const POC_IOT_TICK_TIME: time::Duration = time::Duration::from_secs(1800);
+// TODO: decide some default value, not that it really matters if we set it in the .env
+pub const DEFAULT_LAST_POC_SUBMISSION_TS: i64 = 0000000000;
```

Comment:
I'm still not clear what units this is in? Seconds since epoch?

Reply:
Updated 49d15fd